src/engraving/rendering/dev/systemlayout.cpp

```diff
@@ -789,6 +789,15 @@ void SystemLayout::layoutSystemElements(System* system, LayoutContext& ctx)
         }
     }
 
+    static auto staffOffset = [](EngravingItem* e) -> PointF {
```

Comment:
Repeated from segment.cpp. This should also be replaced by a method in EngravingItem